utils/build.ps1

```diff
@@ -1361,6 +1361,9 @@ function Build-CMakeProject {
             if ($SwiftSDK -ne "") {
               $SwiftArgs += @("-sdk", $SwiftSDK)
             } else {
+              # If you update this, you also need to update test/lit.cfg, which
+              # adds these arguments to config.host_build_swift as well as
+              # putting them into config.*_overlay_opt.
```

Comment:
I wonder if we can embed these into the build system and entirely avoid the addition here.

Reply:
That would be nice, but I don't think that's a change I want to make as part of this PR; I'm just trying to unblock merging this PR.